### ISSUE TITLE
[Feature request] Add cipher benchmark

### ISSUE THREAD

**Issue by Olejja** [hardcore-sushi/DroidFS#235]
I think that a built-in benchmark of the ciphers used would be a useful addition to DroidFS.

**Comment by hardcore-sushi** [comment]
Yes this could be nice to have. In your opinion, where the option should be available?

**Comment by Olejja** [comment]
We have two implementations of encrypted file systems here, so the test will be extensive:

* First gocryptfs with all ciphers
* Then cryfs with all the ciphers

Besides the cipher speed test, I would like to know how long each container implementation takes to create and mount.
I think that a separate item is needed in the settings.
